src/app-layout/visual-refresh-toolbar/toolbar/index.tsx

```diff
@@ -180,7 +180,11 @@ export function AppLayoutToolbarImplementation({
     >
       <div className={styles['toolbar-container']}>
         {hasNavigation && (
-          <nav className={clsx(styles['universal-toolbar-nav'])}>
+          <nav
+            className={clsx(styles['universal-toolbar-nav'])}
+            role={navigationOpen ? 'presentation' : 'navigation'}
+            aria-label={navigationOpen ? undefined : ariaLabels?.navigation}
```

Comment:
To make this code more error-proof, it could be a separate object

```js
const navLandmarkAttributes = navigationOpen ? {role: 'presentation'} : {role: 'navigation', 'aria-label': ariaLabels?.navigation};
```